zerodb/catalog/indexes/field.py

```diff
@@ -1,11 +1,58 @@
 import itertools as it
 
+from persistent import Persistent
+from ZODB.broken import Broken
 from repoze.catalog.indexes.field import CatalogFieldIndex as _CatalogFieldIndex
+from repoze.catalog.indexes.common import CatalogIndex
+from repoze.catalog import RangeValue
 from zerodb import trees
 from zerodb.catalog.indexes.common import CallableDiscriminatorMixin
 from zerodb.storage import prefetch
 from zerodb.util.iter import ListPrefetch
 
+import BTrees
+
+BTreeSet = BTrees.family32.IF.TreeSet
```

Comment:
We should use self.family.IF.TreeSet (in CatalogFieldIndex class) instead because:
* self.family class has bucket size optimal for remote traversal
* family could be family32 or family64, depending on our needs (as opposed to when we hardcode it)